app/assets/scripts/components/documents/pdf-preview/index.js

```diff
@@ -141,13 +167,13 @@ function PdfPreview() {
       {atbd.status === 'succeeded' && (
         <div>
           <div id='content' ref={contentRef}>
-            <Container>
+            <div>
```

Comment:
@frozenhelium just curious why you needed to do this :)

Reply:
We probably don't need this one